test

```diff
@@ -77,6 +90,60 @@ function integration_pass {
 	go test -timeout 1m -v ${RACE} -cpu 1,2,4 -run=Example $@ ${TEST}
 }
 
+function unit_cov {
+	echo "Running code coverage for unit test..."
+	# only -run=Test so examples can run in integration tests
+	# go list (...) | xargs (...) issues following commands where each command runs a unit test for a given pkg and generates its coverage report, e.g $COVEROUT/main.coverprofile for github.com/coreos/etcd
+	# 	bash -c go test -covermode atomic -coverprofile $COVEROUT/main.coverprofile ${RACE} -cpu 1,2,4 -run=Test github.com/coreos/etcd
+	# 	ok  	github.com/coreos/etcd	0.019s	coverage: 0.0% of statements
+	#   ...
+	# unit tests should use atomic covermode to ensure additional coverage logic inserted to the source code by go cover does not induce data race due to the multi-threaded nature of integration tests
```

Comment:
coverage analysis should probably run without race detection and `-cpu 1,2,4`; if it depends on that for getting more coverage, then the tests are flakey. The set covermode should be OK instead of atomic.